component/core/src/main/java/org/exoplatform/social/core/profileproperty/ProfilePropertyServiceImpl.java

```diff
@@ -205,6 +209,9 @@ public void updatePropertySetting(ProfilePropertySetting profilePropertySetting)
         && getUnhiddenableProfileProperties().contains(profilePropertySetting.getPropertyName())) {
       throw new IllegalArgumentException(String.format("%s cannot be hidden", profilePropertySetting.getPropertyName()));
     }
+    if (profilePropertySetting.isMultiValued() && isUserCardFieldSettings(profilePropertySetting.getPropertyName())) {
+      throw new IllegalArgumentException(String.format("%s cannot be multivalued", profilePropertySetting.getPropertyName()));
+    }
```

Comment:
```suggestion
     if (!canEditMultiValued(profilePropertySetting.getPropertyName())) {
      throw new IllegalArgumentException(String.format("Mulativalued property %s cannot be edited", profilePropertySetting.getPropertyName()));
    }
```
Then you should intercept this exception in rest layer to block the edit